src/Lykke.Job.CandlesProducer.AzureRepositories/CandlesGeneratorSnapshotRepository.cs

```diff
@@ -27,11 +27,11 @@ public CandlesGeneratorSnapshotRepository(ILog log, IBlobStorage storage)
             _storage = storage;
         }
 
-        public async Task SaveAsync(ImmutableDictionary<string, ImmutableList<ICandle>> state)
+        public async Task SaveAsync(ImmutableDictionary<string, ICandle> state)
```

Comment:
Don't see here await code

Reply:
May be I've not got you right, but:
https://github.com/LykkeCity/Lykke.Job.CandlesProducer/blob/2c8617f628e3b062fe83ec31dbfe76f4c97b9923/src/Lykke.Job.CandlesProducer.AzureRepositories/CandlesGeneratorSnapshotRepository.cs#L38
and:
https://github.com/LykkeCity/Lykke.Job.CandlesProducer/blob/2c8617f628e3b062fe83ec31dbfe76f4c97b9923/src/Lykke.Job.CandlesProducer.AzureRepositories/CandlesGeneratorSnapshotRepository.cs#L41

Reply:
Sorry, my fail. Did see.